eng/platform-matrix.yml

```diff
@@ -252,8 +252,7 @@ jobs:
       osGroup: Windows_NT
       osIdentifier: Windows_NT
       helixQueues:
-      # TODO: Due to the limited capacity of the machines in Windows.10.Arm64.Open queue limit this to CI builds and only to Windows_NT/arm
-      - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(parameters.helixQueueGroup, 'ci')) }}:
+      - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(parameters.helixQueueGroup, 'pr', 'ci')) }}:
```

Comment:
This will enable using Windows_NT/arm in PRs - is this a temporaty change you are planning to undo before merging?

Reply:
I need this change for any Windows/arm corefx testing to happen. The platforms enabled for a PR are defined in azure-pipelines.yml (or the new yml equivalents). If they don't want Linux_arm, they shouldn't specify it.